dev_docs/older/release_model.html

```diff
@@ -50,7 +50,7 @@
 <p>CVS will not be released as a package and placed on the main
 download site. Users can obtain a copy of the CVS anonymously through
 sourceforge. Instructions for anonymous CVS access for non-developers
-can be found <a href="http://sourceforge.net/cvs/?group_id=44492">here</a>.</p>
+can be found <a href="https://sourceforge.net/projects/triplea/">here</a>.</p>
```

Comment:
Just look at the links to maddlinks.org ^^
This appears to be an old tripleaforum which doesn't even exists anymore. It was captured a couple times on archive.org so I replaced the links with the archive links 😄